Code/GraphMol/ChemTransforms/testChemTransforms.cpp

```diff
@@ -91,6 +91,27 @@ void testDeleteSubstruct() {
   TEST_ASSERT(mol2);
   TEST_ASSERT(mol2->getNumAtoms() == 4);
 
+  // test chiralty
```

Comment:
This test needs an addition to show that both examples are still processed when useChirality is false.